lib/model/CAnnotatedProbability.cc

```diff
@@ -166,6 +166,7 @@ void SAnnotatedProbability::swap(SAnnotatedProbability& other) noexcept {
     s_DescriptiveData.swap(other.s_DescriptiveData);
     std::swap(s_CurrentBucketCount, other.s_CurrentBucketCount);
     std::swap(s_BaselineBucketCount, other.s_BaselineBucketCount);
+    std::swap(s_ShouldUpdateQuantiles, other.s_ShouldUpdateQuantiles);
```

Comment:
I'm pretty sure we should add this to persist and restore as well since we have persistence for this object.